src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,98 @@
+package parser;
+
+import com.github.javafaker.Faker;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JsonParserTest {
+
+    public static final String RESOURCE_PATH = "src/main/resources/%s.json";
+    private JsonParser jsonParser;
+    private static String cartName;
+    private static String filePath;
+    private static Faker faker;
+    private static ArrayList<String> listOfFiles;
+
+    @BeforeEach
+    public void init() {
+        jsonParser = new JsonParser();
+        faker = new Faker();
+        cartName = faker.name().firstName().toLowerCase(Locale.ROOT);
+        filePath = String.format(RESOURCE_PATH, cartName);
+        listOfFiles = new ArrayList<>();
+        listOfFiles.add(filePath);
+    }
+
+    @Disabled("Disabled for testing purposes")
+    @DisplayName(value = "JSON Parser test - Write to existing file")
+    @Test
+    public void ParserWritePositiveTest() {
+        Cart cart = new Cart(cartName);
+        jsonParser.writeToFile(cart);
+
+        File file = new File(filePath);
+        Cart expectedCart = jsonParser.readFromFile(file);
```

Comment:
You was reading from file with your custom code and it was correct. Could you please return this part?
Explanation: when you are developing test on "write" operation, you need to write using JsonParser and read using with any code, but not JsonParser class. This approach guarantees that you will catch bug (if exists) between 2 methods in JsonParser test.